Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN set -ex \
 
 ### Final Stage ###
 
-FROM arti.dev.cray.com/baseos-docker-master-local/alpine:3.13
+FROM artifactory.algol60.net/docker.io/alpine:3.13
```

Comment:
Make this one `artifactory.algol60.net/docker.io/library/alpine:3.13`

Reply:
For context: 
``` Andrew Nieuwsma  7:13 PM
Why that library change out of curiosity? Is the other one bad?

Zach Crisler  7:46 PM
It's actually at the library location -- it's the canonical form (edited) 
7:46
What you had relies in a redirection
```